backend/routers/projects.router.js

```diff
@@ -7,6 +7,8 @@ const { AuthUtil } = require("../middleware");
 // The base is /api/projects
 router.get('/', ProjectController.project_list);
 
+router.put('/', ProjectController.projects);
```

Comment:
Can we add an explanation comment that this request is retruning a PMs project list.